shoop/themes/classic_gray/static_src/js/custom.js

```diff
@@ -29,6 +29,28 @@ function getProductListViewMode() {
     return "grid";
 }
 
+function updateProductPrice(productId) {
+    const $quantity = $("#product-quantity");
+    if (!$quantity.is(":valid")) {
+        return
+    }
+    var quantityParam = '&quantity=' + $quantity.val();
+
+    var childParam = "";
+    const $selectedChild = $("#product-variations");
+    if ($selectedChild.length > 0) {
+        var childParam = "&child=" + $selectedChild.val();
+    }
+
+    $.ajax({
+        url: '/xtheme/product_price?id=' + productId + quantityParam + childParam,
```

Comment:
Instead of constructing the URL with string concatenation, you could use `data` property here.